docs/getting_started/quickstart.md

```diff
@@ -1,29 +1,29 @@
 # Quickstart
 
 ## Installation
-To install PromptGuard, run:
+To install OpaquePrompts, run:
 
 ```bash
-pip install promptguard
+pip install opaqueprompts
 ```
 
 ## Environment Setup
-Accessing the PromptGuard API requires an API key, which you can get by creating an account [on the PromptGuard website](https://promptguard.opaque.co). Once you have an account, you can find your API key on the [API Keys page](https://promptguard.opaque.co/api-keys).
+Accessing the OpaquePrompts API requires an API key, which you can get by creating an account [on the OpaquePrompts website](https://opaqueprompts.opaque.co). Once you have an account, you can find your API key on the [API Keys page](https://opaqueprompts.opaque.co/api-keys).
 
 Once you have your key, set it as an environment variable:
 
 ```bash
-export PROMPTGUARD_API_KEY="..."
+export OPAQUEPROMPTS_API_KEY="..."
 ```
 
-## Using PromptGuard standalone
-PromptGuard offers two main functions: `sanitize()` and `desanitize()`. `sanitize()` takes a string and returns a sanitized (i.e. encrypted and redacted) version of it, while `desanitize()` takes a sanitized string and returns the original string.
+## Using OpaquePrompts standalone
+OpaquePrompts offers two main functions: `sanitize()` and `desanitize()`. `sanitize()` takes a string and returns a sanitized (i.e. encrypted and redacted) version of it, while `desanitize()` takes a sanitized string and returns the original string.
 
 ```python
-import promptguard as pg
+import opaqueprompts
```

Comment:
are we not planning to demonstrate a shorthand for importing like `op`, I think generally speaking you never import a full package like this with this long of a name, you either do `import opaquepromps as op` or something like `from opaqueprompts import sanitize, desanitize`.

Reply:
For writing code, I agree with you. However, I wanted to make it very clear in the documentation what functions are from `opaqueprompts`, so I did not abbreviate the package name.

Reply:
Fair enough, only reason I think abbreviating could be helpful is it could let us establish a "canonical" abbreviation of the package (like np or pd) rather than different people using different abbreviations. No strong opinion however.